bundles/org.openhab.binding.hue/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -15,27 +15,43 @@
 		<item-type>Number:Illuminance</item-type>
 		<label>Illuminance</label>
 		<description>Current illuminance.</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Light</tag>
+		</tags>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
 
 	<channel-type id="light_level" advanced="true">
 		<item-type>Number</item-type>
 		<label>Light Level</label>
 		<description>Current light level.</description>
+		<tags>
+			<tag>Measurement</tag>
```

Comment:
I'd prefer "Measurement" tags to be only used for Numbers with dimension, so that it is clear, what exactly is measured.

Reply:
But what else to use in this case ?

Reply:
Following our agreement to only tag what is obvious, I'd vote to not tag it at all.

Reply:
Ok, this is now removed.